backend/src/test/java/com/woowacourse/levellog/domain/LevellogRepositoryTest.java

```diff
@@ -95,4 +97,29 @@ void notExists() {
             assertFalse(actual);
         }
     }
+
+    @Test
+    @DisplayName("findAllByAuthor 메서드는 주어진 author가 작성한 레벨로그를 모두 반환한다.")
+    void findAllByAuthor() {
+        // given
+        final Member author = memberRepository.save(new Member("pepper", 1111, "pepper.png"));
+        final Member anotherAuthor = memberRepository.save(new Member("roma", 12345, "roma.png"));
+        final Team team = teamRepository.save(
+                new Team("선릉 브라운조", "무중력 광장", LocalDateTime.now().plusDays(1), "campus.png"));
+        final Team team2 = teamRepository.save(
+                new Team("선릉 브라운조", "무중력 광장", LocalDateTime.now().plusDays(1), "campus.png"));
```

Comment:
```suggestion
        final Team team = teamRepository.save(
                new Team("선릉 브라운조", "무중력 광장", LocalDateTime.now().plusDays(1), "campus.png"));
        final Team team2 = teamRepository.save(
                new Team("선릉 네오조", "수성방", LocalDateTime.now().plusDays(1), "campus2.png"));
```

귀찮을 수도 있지만 테스트에서 사용되는 값이 조금씩 다르면 어떨까요? 테스트는 메서드의 명세이기 때문에 실제 데이터같은 값을 넣어주는게 좋다는 리뷰를 받았었어서 생각이 났습니다!